frontend/src/pages/PostPage/components/ReportModal/index.styles.ts

```diff
@@ -0,0 +1,56 @@
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+
+export const ReportModalContainer = styled.form`
+  background-color: white;
+  padding: 1em;
+  box-shadow: 0px 1px 7px ${props => props.theme.colors.gray_150};
+`;
+
+export const Title = styled.p`
+  font-weight: 700;
+  font-size: 1.5rem;
+  margin-bottom: 1rem;
+`;
+
+export const Notice = styled.p`
+  font-weight: 400;
+  font-size: 0.8rem;
+  margin-bottom: 1.8rem;
+  color: ${props => props.theme.colors.gray_200};
+`;
+
+export const Message = styled.textarea`
+  resize: none;
+  width: 15rem;
+  height: 10rem;
+  border: 1px solid ${props => props.theme.colors.gray_50};
+  margin-bottom: 1.5rem;
```

Comment:
```suggestion
  margin-bottom: 1.5rem;
  padding: 10px;
  box-sizing: border-box;
```

글쓰는 area에 padding이 조금 필요한 것 같습니다.